fastlane/spec/cli_tools_distributor_spec.rb

```diff
@@ -33,6 +33,12 @@
   end
 
   describe "update checking" do
+    before(:each) do
+      # prevent the update checker to run and clutter the output
+      # (FastlaneCore::UpdateChecker.start_looking_for_update() will return)
+      ENV["FASTLANE_SKIP_UPDATE_CHECK"] = "a_truthy_value"
```

Comment:
I'm wondering if this doesn't risk modifying the `ENV` of the user that is running the test suite?
Maybe better to `allow(ENV).to receive(:[]).with('FASTLANE_SKIP_UPDATE_CHECK').and_return('a_truthy_value')` instead?

Reply:
Thanks. That was indeed helpful. I've added 8840cf664 which restores the context by using our `ClimateControl` wrapper. Using allow is harder as you have to take into account whether ENV is accessed many times.